module-example/package.json

```diff
@@ -0,0 +1,28 @@
+{
+    "name": "powerhouse-switchboard-plugin-demo",
+    "version": "0.0.1",
+    "description": "Open-source API over database models made to be developer- and analyst-friendly.",
+    "author": "makerdao",
+    "type": "module",
+    "license": "AGPL-3",
+    "scripts": {
+        "dev": "vite-node -w src/index.ts",
+        "start": "NODE_ENV=production vite-node src/index.ts",
+        "debug": "DEBUG=1 npm run dev",
+        "lint": "eslint \"./**/*.{ts,tsx}\" --max-warnings=0",
+        "typecheck": "tsc --noEmit"
+    },
+    "dependencies": {
+        "@prisma/client": "^4.11.0",
+        "ethers": "^5.7.2",
+        "graphql": "^16.7.1",
+        "nexus": "^1.3.0",
+        "siwe": "^2.1.4"
+    },
+    "devDependencies": {
+        "@types/node": "^18.14.2",
+        "eslint": "^8.35.0",
+        "prisma": "^4.11.0",
```

Comment:
Why do you need `prisma`, `@prisma/client`, `graphql`, `ethers` and `siwe` in the package? Do you use/import anything from there?

Reply:
cleaned up